internal/config/operations.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"strings"
+
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/meshes"
+	smp "github.com/layer5io/service-mesh-performance/spec"
+)
+
+var (
+	CiliumOperation          = strings.ToLower(smp.ServiceMesh_OPEN_SERVICE_MESH.Enum().String())
+	CiliumBookStoreOperation = "cilium_bookstore_app"
+	ServiceName              = "service_name"
+)
+
+func getOperations(op adapter.Operations) adapter.Operations {
+	versions, _ := getLatestReleaseNames(3)
+
+	op[CiliumOperation] = &adapter.Operation{
+		Type:        int32(meshes.OpCategory_INSTALL),
+		Description: "Open Service Mesh",
```

Comment:
Please remove  "open service mesh" from everywhere

Reply:
Made the changes 👍